tests/core/switch_session_test.py

```diff
@@ -17,16 +17,16 @@
 
 from flexmock import flexmock
 from hamcrest import assert_that, is_
-import mock
-
+from mock import Mock, patch
 from netman.core.objects.exceptions import UnknownResource, \
-    OperationNotCompleted, NetmanException, SessionAlreadyExists
+    OperationNotCompleted, NetmanException, SessionAlreadyExists, UnknownSession
 from netman.core.switch_sessions import SwitchSessionManager
 
 
 class SwitchSessionManagerTest(TestCase):
     def setUp(self):
         self.switch_mock = flexmock()
+        self.switch_mock.switch_descriptor = "I describe switches"
```

Comment:
I would advise to use real objects here  SwitchDescriptor("cisco", "host") instead of an INT.

If the code ever wants to use functionnalities of this object it willl be able to and it's always less misleading to anybody who reads the test and wonder what the heck is this variable in this weakly-typed language